frontend/src/photoView/photoView.js

```diff
@@ -11,11 +11,135 @@ const SIDES = {
     BINDER: 'binder',
 };
 
+export class FVDisplayWidget extends React.Component {
+    render() {
+        const items = [];
+        let line;
+        for (line of this.props.lineCoords) {
+            line['1_y'] = (line['1_y'] * this.props.height) / this.props.natHeight;
+            line['2_y'] = (line['2_y'] * this.props.height) / this.props.natHeight;
+            line['1_x'] = (line['1_x'] * this.props.width) / this.props.natWidth;
+            line['2_x'] = (line['2_x'] * this.props.width) / this.props.natWidth;
+            items.push(<line
+                x1={line['1_x']}
+                y1={line['1_y']}
+                x2={line['2_x']}
+                y2={line['2_y']}
+            />);
+        }
+        if (this.props.vanishingPointCoord !== null) {
+            items.push(<circle
+                cx={(this.props.vanishingPointCoord.x * this.props.width) / this.props.natWidth}
+                cy={(this.props.vanishingPointCoord.y * this.props.height) / this.props.natHeight}
+                r='10'
+            />);
+        }
+        return (
+            <div>
+                <svg
+                    className='analysis-overlay floatTL'
+                    height={this.props.height}
+                    width={this.props.width}
+                >
+                    {items}
+                </svg>
+            </div>
+        );
+    }
+}
+
+function configAnalysisFV(parsedValue, height, width, natHeight, natWidth) {
```

Comment:
not gonna belabor this, bc we're talking this through in person. but you get the point.